plugins/woocommerce/.gitignore

```diff
@@ -34,3 +34,6 @@ i18n/languages/woocommerce.pot
 # Allure directories
 allure-report
 allure-results
+
+# Work in progress performance test
+**/tests/performance/**/wip.js
```

Comment:
Thanks for raising this, @tammullen! This is something I wanted to discuss.

I had to add a new entry to the changelog (otherwise I wasn't able to commit the changes). As you suggested, I think this occurs because of the `.gitignore` change, but I'm not 100% sure. I just compared my changes with other previous performance tests related PRs and found out that this was the only difference.

I just wanted to add a new scenario named `wip.js`, so that we could test all the requests we needed without having to modify other real scenarios. And I wanted to add it to the `.gitignore` in order to avoid committing it by mistake.

Do you think this could be a good approach for running and debugging new tests? In that case, I would update the README with the following instructions to contribute to a new test:
```
1. Create the requests you need for your shopper and/or merchant flows at plugins/woocommerce/tests/performance/requests.
2. Use the required requests and build your user flow in plugins/woocommerce/tests/performance/tests/wip.js.
3. Run your scenario with k6 run tests/wip.js
4. Once you can confirm your new requests and flows work properly, you can integrate them with an existing test or create a new one.
5. Commit changes.
```

This way, we would be able to suggest a common workflow when adding new requests and tests, and this way I think we might make it easier to start contributing to this project.

Do you think it would be a good idea doing this? Otherwise, I could revert the `.gitignore` and changelogger changes.

Thanks!